web/src/main/webapp/WEB-INF/jsp/login/login.jsp

```diff
@@ -1,8 +1,9 @@
 <%@page contentType="text/html;charset=UTF-8" language="java" %>
 <%@ include file="/WEB-INF/jsp/taglibs.jsp" %>
+<%String action = request.getParameter("action");%>
 <!-- For Mantis Issue 6099 -->
 <jsp:useBean scope='session' id='userBean' class='org.akaza.openclinica.bean.login.UserAccountBean'/>
-    <c:if test="${userBean.name!=''}">
+    <c:if test="${userBean.name!='' && action!=null}">
```

Comment:
it seems like the check should be "action == null". Please confirm

Reply:
done